### PR TITLE
Added index.html,style.css and app.js files for the pop up window

### PR DESCRIPTION
**Pull request by iankiema**
The following files have been added for the pop up window:
Index.html
style.css
app.js

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -26,122 +27,107 @@ mobLink.addEventListener('click', () => { topNav.style.display = 'none'; });
...
-// //   card.classList.add('project-card');
-// //   card.innerHTML = `
-// //     <img src="${project.image}" alt="Project Image">
-// //     <h3>${project.name}</h3>
-// //     <button class="details-btn">Details</button>
-// //   `;
-// //   return card;
-// // }
+showCaseBtns.forEach((showCaseBtn) => {
+  // eslint-disable-next-line func-names
```
Comment (mikethreels) [inline app.js]: - [ ] the linters are used to make your project work better and easier to read, therefor please avoid turning of any linters.

File: index.html [hunk 2]
```diff
@@ -95,7 +95,7 @@ <h2>Multi-Post Stories</h2>
 
             <div class="showcase" id="showcase">
                 <div class="showcase-image">
-                   <img src="images/Img Placeholder (3).png" alt="">
+                   <img src="images/Img Placeholder (3).png" alt="project image">
                 </div>
 
                 <div class="showcase-detail">
```
Comment (mikethreels) [inline index.html]: - [ ] the project in your recent works sections should be loaded dynamically using JS code, right now the hardcoded HTML protects are still displayed on the webpage and that's probably the reason the pop-up isn't working either.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Cizzla, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job making a start with the JS code.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -91,124 +92,44 @@ <h2>Multi-Post Stories</h2>
...
-                  <h2>Profesional Art Printing Data</h2>
-                  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
-                  <ul>
-                      <li>HTML</li>
-                      <li>Bootstrap</li>
-                      <li>Ruby</li>
-                  </ul>
-              </div>
-              <button id="showcase-button"  class="showcase-btn" type="button">See project</button>
+    <div id="popup" class="popup">
```
Comment (shahadat3669) [inline index.html:86]: - [ ] Well done! 👏🎉 It seems you have made progress in addressing the previous reviewer's feedback and resolving the linter errors. However, there are still a few adjustments that need to be made. 

- [ ] First, as mentioned by the previous reviewer, the projects in your "Recent Works" section should be loaded dynamically using JavaScript code. This means that not only the secondary projects but also the main project should be loaded dynamically from the array. Currently, it seems that the main project is not being loaded dynamically. Please update your code to ensure that the main project is also loaded dynamically.

- [ ] Second, the pop-up content should also be populated dynamically. Based on the current state of your code, it appears that the pop-up content is not being dynamically generated based on which project the user click. Please make the necessary changes to ensure that the pop-up content is dynamically populated as required.

[Ian-Kiema-Portifolio.webm](https://github.com/iankiema/My-Portifolio-Personal/assets/55840999/bdb0c2fb-6fff-4710-8e30-d6ff13edc2b4)

Please review these points and update your code accordingly. If you have any questions or need further assistance, feel free to ask. Keep up the good work! 💪🌟

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi Everyone, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
☑️ Good PR Title
☑️ Good PR description
☑️ Good use of GitHub Flows

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @iankiema, @redtah,

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added main page and pop-up dynamically. ✔️ 

## [Optional] suggestions:
- For future pull requests, please notice that your branch name `windows-popup` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-windows-popup`. This makes it easier to track the changes and the purpose of each branch.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**